R/ard_categorical.R

```diff
@@ -104,7 +108,7 @@ ard_categorical.data.frame <- function(data,
                                        by = dplyr::group_vars(data),
                                        strata = NULL,
                                        statistic = everything() ~ c("n", "p", "N"),
```

Comment:
Since `n_cum` and `p_cum` are not part of the default set of stats, are they referenced somewhere else in the documentation?

Reply:
omg, yes, thank you so much for the thoughtful review! I just added `c("n", "N", "p", "n_cum", "p_cum")`. Do you think that is enough?

Reply:
yes!